package.json

```diff
@@ -131,6 +131,10 @@
       "path": "packages/client/runtime/edge-esm.js",
       "gzip": false
     },
+    {
+      "path": "packages/client/runtime/wasm.js",
+      "gzip": false
+    },
```

Comment:
No need of a `wasm-esm.js`?

Reply:
Not yet, one day when we get back to esm